science/shtools/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                shtools
+revision            0
+
+categories          science math
+platforms           darwin
+license             BSD-3-Clause
+maintainers         {@MarkWieczorek gmail.com:mark.a.wieczorek}
+description         Spherical Harmonic Tools
+long_description    SHTOOLS is a library of Fortran 95 software that can be \
+                    used to perform spherical harmonic transforms, multitaper \
+                    spectral analyses, expansions of functions into Slepian \
+                    bases, and standard operations on global gravitational \
+                    and magnetic field data. Requires linking with FFTW-3 and \
+                    LAPACK compatible libraries.
+homepage            https://shtools.github.io/SHTOOLS/
+
+github.setup        SHTOOLS SHTOOLS 4.7.1 v
+github.tarball_from   archive
+checksums           sha256  176610743ddd76ea1fe7b45db3e8613171630b85c3759c9a4b31cde6b5d2ae3d \
+                    rmd160  58959a91c2889488844f6a7e4745090b0c906057 \
+                    size    35165653
+
+use_configure       no
+
+build.cmd           make
+build.target        fortran F95="${prefix}/bin/gfortran-mp-10"
+
+variant openmp description {Add OpenMP support} {
+    use_parallel_build      no
+    build.target-append     fortran-mp
+}
+
+test.run            yes
+test.cmd            make
+test.target         fortran-tests-no-timing
+test.args           PREFIX=${prefix} F95="${prefix}/bin/gfortran-mp-10" LAPACK="-framework Accelerate" BLAS=""
+
+destroot.cmd        make
+destroot.target     install
```

Comment:
I think you can remove the lines beginning build.cmd, test.cmd, destroot.cmd and destroot.target, because the values you've specified are the default values.